### PR TITLE
feat(EU):  Vehicle2Load Discharge Limit Sensors

### PR DESCRIPTION
**Pull request by jwillemsen**
Expose Vehicle2Load Discharge limit, currently no set is supported, but the value is now shown correctly in HA

### CODE REVIEW

**Comment by jwillemsen** [comment]
https://github.com/Hyundai-Kia-Connect/kia_uvo/issues/819

File: custom_components/kia_uvo/number.py [hunk 1]
```diff
@@ -103,14 +113,22 @@ async def async_set_native_value(self, value: float) -> None:
...
 
         # set new limits
         if self._description.key == AC_CHARGING_LIMIT_KEY:
             ac = value
             dc = self.vehicle.ev_charge_limits_dc
-        else:
+        if self._description.key == DC_CHARGING_LIMIT_KEY:
             ac = self.vehicle.ev_charge_limits_ac
             dc = value
+        if self._description.key == V2L_DISCHARGE_LIMIT_KEY:
```
Comment (cdnninja) [inline custom_components/kia_uvo/number.py:129]: If the above now filters to the key does this need to exist? Haven't looked closely at the code though.  

My impression is this if statement doesn't do anything.

Comment (jwillemsen) [reply custom_components/kia_uvo/number.py:129]: Code above guards setting the same value, in this place later we can add the code to set the value to the lower layer

Comment (Bubo08) [reply custom_components/kia_uvo/number.py:129]: I think the code is correct and could be merged. 

I would prefere the use of elif which makes it more readable but in the code above it is also not used.

**Comment by fuatakgun** [comment]
> Expose Vehicle2Load Discharge limit, currently no set is supported, but the value is now shown correctly in HA

if set is not supported, let's expose it as SENSOR rather than NUMBER

**Comment by jwillemsen** [comment]
> > Expose Vehicle2Load Discharge limit, currently no set is supported, but the value is now shown correctly in HA
> 
> if set is not supported, let's expose it as SENSOR rather than NUMBER

Hopefully we can make it supported soon (currently not having enough time), making it now a number and change it in some weeks to a sensor leads to an unavailable sensor at that moment.

**Comment by fuatakgun** [comment]
I don't think it is a big thing to worry about?

**Comment by jwillemsen** [comment]
Ok, will create a new branch with that idea asap

**Pull request closed without merging**